### DISCUSSION TITLE
File picker?

### DISCUSSION THREAD

**Discussion by carlbeech** [beeware/toga#1550] [Q&A]
Hi

Firstly apologies if this is a newbee question and I've missed something in the docs...

I'm looking to port an app I've created in python/QT5 over to toga so that I can use it in both windows and Android - however I've not spotted a 'file picker' dialog - can someone point me to an example piece of code where I can do that?  (e.g. either a single standard call to a function, or if I've got to check the platform and call different file-picker functions...?)

Many thanks

Carl.

**Comment by mhsmith** [comment]
This has been previously discussed (#1158 and #1171), but Toga doesn't currently provide any cross-platform file picker API. So you'll have to write separate code for each platform.

On Android, you can use the `intent_result` method which was added in #1217.

**Reply by carlbeech** [reply]
Ok - hmmm sounds quite complex from what I've read (I've not dealt with Android programming before... I'll have to have a play to see...

Many thanks for the swift reply :-)
